README.md

```diff
@@ -215,6 +216,7 @@ Python plugins developers must ensure their plugin to work with all Python versi
 [kotlin-example]: https://vincenzopalazzo.medium.com/a-day-in-a-c-lightning-plugin-with-koltin-c8bbd4fa0406
 [Lightning-Invoice-Queue]: https://github.com/rbndg/Lightning-Invoice-Queue
 [lightning-qt]: https://github.com/darosior/pylightning-qt
+[ln-address-pay]: https://github.com/nosedam/ln-address-pay
```

Comment:
Can you move this down one line? It's not in alphabetical order if it's above listmempoolfunds.

Reply:
Done and squashed!